Extensions/TextInput/textinputruntimeobject-pixi-renderer.ts

```diff
@@ -119,6 +119,8 @@ namespace gdjs {
       if (!this._input) return;
       this._input.remove();
       this._input = null;
+      this._form?.remove();
```

Comment:
I think you should add the check just as above, making sure that each variable is defined before calling any method on it.